scheduler/src/cook/kubernetes/api.clj

```diff
@@ -777,15 +785,19 @@
       (assert (= pod-name-from-pod pod-name)
               (str "Pod name from pod (" pod-name-from-pod ") "
                    "does not match pod name argument (" pod-name ")"))
-      (log/info "Launching pod with name" pod-name "in namespace" namespace ":" (Yaml/dump pod))
+      (log/info "In" compute-cluster-name "launching pod with name" pod-name "in namespace" namespace ":" (Yaml/dump pod))
       (try
-        (create-namespaced-pod api namespace pod)
+        (timers/time! (metrics/timer "launch-pod" compute-cluster-name)
+          (create-namespaced-pod api namespace pod))
         true
         (catch ApiException e
           (let [code (.getCode e)
                 bad-pod-spec? (contains? #{400 404 422} code)]
-            (log/info e "Error submitting pod with name" pod-name "in namespace" namespace
+            (log/info e "In" compute-cluster-name "error submitting pod with name" pod-name "in namespace" namespace
```

Comment:
Same comment as above about the pattern for these logs.

Reply:
Changed